.github/ISSUE_TEMPLATE/service.md

```diff
@@ -1,6 +1,12 @@
-To help us debug your issue please explain:
+---
+name: 'Service: Infrastructure Bugs Or Requests'
```

Comment:
I don't like this "service" option. By definition, it is not the right place to report, probably.

Reply:
I've removed service from the categories.